clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -646,9 +646,69 @@ RValue CIRGenFunction::buildLoadOfLValue(LValue LV, SourceLocation Loc) {
 
   if (LV.isSimple())
     return RValue::get(buildLoadOfScalar(LV, Loc));
+
+  if (LV.isVectorElt()) {
```

Comment:
Seems like this is content from the other patch? If there are dependencies between the PRs, it might be better not to put them up for review just yet, otherwise I have to go to the same thing twice and mentally account for it :)